src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -27,6 +27,14 @@
       String template,
       String outputPath,
       TargetLanguage language) {
+    // Note that internally, this is overridable using a "discovery" section in the service config,
```

Comment:
discovery is not part of the public service config, so let's not mention it here.